apps/style/netsim/style.scss

```diff
@@ -55,6 +55,55 @@ $wide-column-width: 121px;
   left: 0;
   right: 0;
 
+  // Alerts
+
+  .netsim-alert-container {
+    position: absolute;
+    top: -15px;
+    left: 50%;
+    width: 450px;
+    margin-left: -225px; // -(width/2)
+    z-index: 99999;
```

Comment:
This might be a good moment to break out a `layers.scss` file or something.  I wasn't sure whether 99,999 had special significance, but it looks like it might; it shows up just underneath the `.project_more_popup`

From application.css.scss:
```sass
.user_options, .project_more_popup {
  z-index: 100001;
}
```

Don't stop to fix all of our layering problems right now :grin: but want to make sure this number was a mindful choice.